test/distributed/org/apache/cassandra/distributed/test/guardrails/GuardrailPartitionSizeTest.java

```diff
@@ -75,12 +75,11 @@ protected Cluster getCluster()
     @Test
     public void testPartitionSize()
     {
+        // test yaml-loaded config
         testPartitionSize(WARN_THRESHOLD, FAIL_THRESHOLD);
-    }
+        schemaChange("DROP TABLE %s");
 
```

Comment:
one more question : why merge the two test case into one ?"test yaml-loaded config" and "test dynamic config"
as https://github.com/apache/cassandra/commit/b0aa44b27da97b37345ee6fafbee16d66f3b384f described 
there are two cases

Reply:
The tests extending `o.a.c.distributed.test.guardrails.GuardrailTester` use a single cluster for the entire suite. The regular test case uses the yaml configuration set at startup, the one that is set at `setupCluster`. In contrast, the test for dynamic update changes that config on the running instance by simulating the JMX methods.

That works if the test for static config runs before the test for dynamic update, but not the other way around. If the test for dynamic config runs first, the test for static config will find the dynamically updated config, breaking the test. We could simply restart the node before each test, but I think the test will save resources if we simply combine both tests. I think the tests are brief enough to remain readable.

So `GuardrailPartitionSizeTest` only worked incidentally because the tests were run in the fortunate order. The new `GuardrailPartitionTombstonesTest` however was not so fortunate, exposing the bug. So I have changed both to make them independent of whatever order the test runner decides for us.

Reply:
Thank you for your answer